apps/meteor/client/views/marketplace/AppDetailsPage/AppDetailsPageHeader.tsx

```diff
@@ -48,7 +48,7 @@ const AppDetailsPageHeader = ({ app }: { app: App }): ReactElement => {
 					</Box>
 					<Box is='span'> | </Box>
 
-					<Box mi='x16' marginInlineEnd='x4'>
+					<Box mi='x16' marginInlineEnd='x16'>
```

Comment:
```suggestion
					<Box mi='x16'>
```
`mi='x16' is already setting the margin inline **start** and **end** to `x16`